doc/source/reference/routines.ma.rst

```diff
@@ -409,3 +423,4 @@ Miscellanea
    ma.ediff1d
    ma.indices
    ma.where
+   ma.unique
```

Comment:
I think this can go in the `Inspecting the array` section.

Reply:
@Mukulikaa  how can I make these changes? do I need  to go back to the forked repo and change these line and make another PR kindly guide me I am new in PR.

Reply:
You don't need to make a new PR every time. You only need to commit your changes to the branch from which you made your PR.